CHANGELOG.md

```diff
@@ -4,9 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.12.2] - 2024-01-31
+### Changed
+- Added a safety to give a second chance to transmit video in case the video track wasn't fully initialized at room join time. (#106)
```

Comment:
Needs updating.

Reply:
yep! as I was testing my other fix yesterday, I found another issue and thought I'd add it early this morning. I'm on this.